Sources/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -284,6 +288,11 @@ struct MessageRepliesPayload: Decodable {
     let messages: [MessagePayload]
 }
 
+/// An object describing pinned messages JSON payload.
+struct PinnedMessagesPayload: Decodable {
```

Comment:
Why do we have duplicated structs with different names?

Reply:
Yep, we could reuse `MessageRepliesPayload` but having this name in terms of pinned messages would look weird. The option was to rename `MessageRepliesPayload -> MessageListPayload` but this would cause changes in other places.

While writing the reply have got the idea 💡. We can rename `MessageRepliesPayload -> MessageListPayload` and leverage type aliases to have meaningful names. WDYT @polqf?

Reply:
✅ Added type aliases